static/bidder-info/adquery.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://bidder.adquery.io/prebid/bid"
+maintainer:
+  email: prebid@adquery.io
+#endpointCompression: gzip # disabled because otherwise bidder responds with {data:null}
+gvlVendorID: 902
+capabilities:
+#  app: # disabled because currently it's only a site, not an app (?)
+#    mediaTypes:
+#      - banner
+  site:
+    mediaTypes:
+      - banner
+userSync:
+  redirect:
+    url: https://bidder.adquery.io/prebid/userSync?1=1&gdpr={{.GDPR}}&consent={{.GDPRConsent}}&ccpa_consent={{.USPrivacy}}&redirect={{.RedirectURL}}
+    userMacro: $UID
```

Comment:
usersync url does not redirect

```
https://bidder.adquery.io/prebid/userSync?1=1&gdpr=&consent=&ccpa_consent=&redirect=https://ib.adnxs.com/prebid/setuid?bidder=adquery&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=i&uid=$UID
```


<img width="1702" alt="image" src="https://github.com/prebid/prebid-server/assets/24757781/cb70adcb-b683-4da1-bb18-c4ecde3d9791">

Reply:
Bidder logic has been updated, it should redirect now

Reply:
@kamilmclearcode  user id is not included in redirect url

Reply:
@onkarvhanumante should it be the user id taken somewhere from the request (if so, from where?), or user id generated by adquery bidder?

Reply:
@onkarvhanumante done

Reply:
@kamilmclearcode The user sync URL is not working. Could you please re-check?
<img width="1506" alt="Screenshot 2023-08-22 at 12 47 23 PM" src="https://github.com/prebid/prebid-server/assets/8147531/ea0fbf49-3387-408f-9cec-8c524982850c">

Reply:
@gargcreation1992 
What was the response you got?

![image](https://github.com/prebid/prebid-server/assets/125450079/ea49604f-618e-4a5c-a4b8-bc2099836451)

It seems to work fine for me, check this address:
`https://bidder.adquery.io/prebid/userSync?1=1&gdpr=&consent=&ccpa_consent=&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dadquery%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24UID`

Reply:
It seems to be working fine now, thanks!